jupyter-docker/Dockerfile

```diff
@@ -111,7 +111,8 @@ RUN echo "deb $DEBIAN_REPO/debian jessie main\ndeb $DEBIAN_REPO/debian-security
 ADD spark-defaults.conf.template $SPARK_HOME/conf/
 # Spark defaults conf find and replace [HAILHASH] with the hash
 RUN sed "s|\[HAILHASH\]|$HAILHASH|g" $SPARK_HOME/conf/spark-defaults.conf.template > $SPARK_HOME/conf/spark-defaults.conf
-ADD jupyter_notebook_config.py $JUPYTER_HOME
+ADD jupyter_notebook_config.py $JUPYTER_HOME/
```

Comment:
flipside to removing the slash from `$JUPYTER_HOME`: you have to add it here or Docker will think you're trying to add a file